cypress/integration/events.js

```diff
@@ -1,58 +1,55 @@
 describe('Event sign up', () => {
   var email
+  var firstName
```

Comment:
Still think these should be `let` following best practice, Cypress actually say to not do this at all though https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases 🤔

Reply:
Ah I didn't realise they had a mechanism for doing this; I'll update it

Reply:
I had a stab at reworking this but I wasn't getting anywhere, I think its a scoping issue but even when I change the arrow functions to be `function()` it still didn't like it for some reason. I'm going to leave it as-is to get this in and unblock the pipeline and we can take a look at reworking it to use aliases later on.